docs/tutorial-modules/pimax.mdx

```diff
@@ -0,0 +1,47 @@
+# VRCFTPimaxModule (Pimax)
+
+## Summary
+
+The VRCFTPimaxModule contains the runtime used to interface with Pimax eye tracking hardware and software on Windows PCs.
+
+## Set-up
+
+Setup of the Pimax Module is fairly simple, though can be a little confusing depending on your hardware/software configuration. Please read carefully to avoid any issues. 
+
+:::note
+This guide assumes you have PiTool and ASeeVR installed and updated prior to using the VRCFTPimaxModule. 
+If you do not have the [PiTool](https://pimaxcom.myshopify.com/pages/pitool-download) and/or [ASeeVR](https://drive.google.com/file/d/1ELDtOnMa-MkgchmWFf7w5an-iPOFtQL8/view?ref=Nebo&_ga=2.157958394.1121053981.1651257342-354291550.1593067405)
```

Comment:
The ASeeVR link denies public access to download

Reply:
Resolved in #08c40db. Changed this link to Guppy's Github